pkg/controller/controller.go

```diff
@@ -130,6 +130,10 @@ func (c *Controller) processItem(pvc *v1.PersistentVolumeClaim) error {
 	if secretName == "" {
 		glog.Infof("processItem: no secret will be supplied to endpoint %q\n", ep)
 	}
+	if metav1.HasAnnotation(pvc.ObjectMeta, AnnImportPod) {
```

Comment:
The `glog` on L87 should be moved to below L90.

Reply:
If the cdi controller operates in its own goroutine and there is only 1 cdi controller running (_threadiness==1), why do we have to worry about a race cond here?